modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,187 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-network-issues.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-manifest-file-multiple-interfaces-br-ex_{context}"]
+= Creating an additional OVN-Kubernetes `br-ex` bridge network
+
+You can create an additional `br-ex` bridge network, `br-ex1`, that has a different configuration to the default `br-ex` bridge. You must define the additional bridge in the target interface definition of a machine config manifest file.  
+
+After you create and edit the manifest file, the Machine Config Operator completes tasks in the following order:
+
+* Drains nodes in singular order based on the selected machine config pool.
+* Injects Ignition configuration files into each node, so that each node receives the additional `br-ex` bridge network configuration.
+* Restarts the node with the modified override file.
+* Executes the `configure-ovs.sh` shell script that references the new interface definition.
+* Adds `br-ex` and `br-ex1` to the host node.
+* Uncordons the nodes. After all the nodes return to the `Ready` state, the Operator applies the `k8s.ovn.org/l3-gateway-config` annotation to each node.
+
+For more information about useful situations for the additional `br-ex` bridge and a situation that always requires the default `br-ex` bridge, see "Configuration for a localnet topology".
+
+.Procedure
+
+. Optional: Create an interface connection that your secondary interface, `br-ex1`, can use by completing the following steps. The steps use example configurations to show the use of two interfaces instead of a machine config manifest file to define a secondary bond interface for your cluster. 
++
+[IMPORTANT]
+====
+Do not use the Kubernetes NMState Operator to define or a `NodeNetworkConfigurationPolicy` (NNCP) manifest file to define the secondary interface.
+
+Also ensure that the secondary interface or sub-interfaces when defining a `bond` interface are not used by existing an `br-ex` OVN-kubernetes network deployment.
+====
++
+.. Create the following interface definition files. These files get added to a machine config manifest file so that host nodes can access the definition files. 
++
+.Example of the first interface definition file that is named `eno1.config`
+[source,ini]
+----
+[connection]
+id=eno1
+type=ethernet
+interface-name=eno1
+master=bond0
+slave-type=bond
+autoconnect=true
+autoconnect-priority=20
+----
++
+.Example of the second interface definition file that is named `eno2.config`
+[source,ini]
+----
+[connection]
+id=eno2
+type=ethernet
+interface-name=eno2
+master=bond0
```

Comment:
bond0 should be renamed bond1

Reply:
:heavy_check_mark: 

`master` and `slave` are words that are phased out of general content. Do other names exist for this components or are they embedded as is in code?